src/components/visualizations/chart/highcharts/commonConfiguration.ts

```diff
@@ -97,23 +102,29 @@ const BASE_TEMPLATE: any = {
 };
 
 function registerDrilldownHandler(configuration: any, chartOptions: any, drillConfig: any) {
-    set(configuration, "chart.events.drilldown", function chartDrilldownHandler(event: any) {
+    set(configuration, "chart.events.drilldown", function chartDrilldownHandler(
+        event: Highcharts.DrilldownEventObject,
+    ) {
         chartClick(drillConfig, event, this.container, chartOptions.type);
     });
 
     return configuration;
 }
 
-function registerLoadHandler(configuration: any, chartOptions: any) {
+export function handleChartLoad(): void {
+    setupDrilldown(this);
+}
+
+function registerRenderHandler(configuration: any, chartOptions: any) {
     if (isOneOfTypes(chartOptions.type, supportedDualAxesChartTypes)) {
-        set(configuration, "chart.events.load", handleChartLoad);
+        set(configuration, "chart.events.render", handleChartLoad);
```

Comment:
Could you please explain why we change 'load' to 'render'? Thanks

Reply:
on event load, `ddPoints` is not available on Axis. `ddPoints` is set on 'render' event. See detail [here](https://github.com/highcharts/highcharts/blob/master/js/modules/drilldown.src.js#L821)